### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sleepingpengwing**
Requesting to pull frontend app changes onto main branch

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -12,22 +12,39 @@ import { useState } from "react";
 
   3. Use function in app
 */
-
-const Button = (): JSX.Element => {
+const Footer = () => {
+  return (
+    <div>
+      greeting app created by <a href='https://github.com/mluukkai'>mluukkai</a>
```
Comment (Hi-Van) [inline frontend/src/App.tsx]: should be your own github link

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -12,22 +12,39 @@ import { useState } from "react";
...
-      }}
+      } }
     >
       Count is {count}{" "}
     </button>
   );
-};
-
+}
+const Hello = (props: { name: string | number | boolean | ReactElement<any, string | JSXElementConstructor<any>> | Iterable<ReactNode> | ReactPortal | null | undefined; age: string | number | boolean | ReactElement<any, string | JSXElementConstructor<any>> | Iterable<ReactNode> | ReactPortal | null | undefined; }) => {
```
Comment (Hi-Van) [inline frontend/src/App.tsx]: please dont do this :(

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -12,22 +12,39 @@ import { useState } from "react";
...
+  console.log(props)
```
Comment (Hi-Van) [inline frontend/src/App.tsx]: remove console logging

File: frontend/src/App.tsx (hunk 1 above)
Comment (sleepingpengwing) [reply frontend/src/App.tsx]: Did that shii

**Review by Hi-Van — APPROVED** [review]
lgtm!

**Pull request merged**